pkg/cgroups/cgroups_supported.go

```diff
@@ -28,11 +29,15 @@ var (
 // IsCgroup2UnifiedMode returns whether we are running in cgroup 2 cgroup2 mode.
 func IsCgroup2UnifiedMode() (bool, error) {
 	isUnifiedOnce.Do(func() {
+		// Do something here
 		var st syscall.Statfs_t
 		if err := syscall.Statfs("/sys/fs/cgroup", &st); err != nil {
 			isUnified, isUnifiedErr = false, err
 		} else {
 			isUnified, isUnifiedErr = st.Type == unix.CGROUP2_SUPER_MAGIC, nil
+			if !isUnified {
+				logrus.Warnf("WARNING! You are currently using cgroups-v1 which is not supported with Podman 5. Consider switching to cgroups-v2.")
```

Comment:
CRI-O is using this code as well, so a Podman related warning may be inappropriate:

https://github.com/cri-o/cri-o/blob/2a20780b7b91fecb27c2663558bb06cc7f817d14/internal/config/node/cgroups_linux.go#L31